src/Tools/DTO/FunctionDeclaration.php

```diff
@@ -38,9 +38,9 @@ class FunctionDeclaration implements WithJsonSchemaInterface
      *
      * @param string $name The name of the function.
      * @param string $description A description of what the function does.
-     * @param mixed $parameters The JSON schema for the function parameters.
+     * @param mixed|null $parameters The JSON schema for the function parameters.
```

Comment:
Isn't `null` a subset of `mixed`?

Reply:
Oops! I did fix this but forgot to push. I'll fix this in the next PR.